torch/csrc/distributed/rpc/process_group_agent.cpp

```diff
@@ -591,10 +590,14 @@ bool ProcessGroupAgent::handleRecv(RecvWork& work) {
     futureCV_.notify_all();
     --clientActiveCalls_;
     if (message.type() == MessageType::EXCEPTION) {
-      fm->setError(
-          std::string(message.payload().begin(), message.payload().end()));
+      //fm->setError(
+      //    std::string(message.payload().begin(), message.payload().end()));
```

Comment:
Any reason the old code is kept around as a comment?

Reply:
ahh, sorry, that's my debug vestige. removing